plugins/python-build/share/python-build/pypy3.5-5.8.0

```diff
@@ -0,0 +1,18 @@
+case "$(pypy_architecture 2>/dev/null || true)" in
+"linux64" )
+  if require_distro "Ubuntu 14.04" 1>/dev/null 2>&1; then
+    install_package "pypy3-v5.8.0-linux64" "https://bitbucket.org/pypy/pypy/downloads/pypy3-v5.8.0-linux64.tar.bz2#57d871a7f1135719c138cee4e3533c3275d682a76a40ff668e95150c65923035" "pypy" verify_py35 ensurepip
+  else
+    install_package "pypy3.5-5.8-beta-linux_x86_64-portable" "https://bitbucket.org/squeaky/portable-pypy/downloads/pypy3.5-5.8.0-beta-linux_x86_64-portable.tar.bz2#76c2c4b2846e38d2b4846d4b5f9dc0bd9fd7a9b1f04beb92975ae090ad390d75" "pypy" verify_py35 ensurepip
```

Comment:
It seems that the download URL need to be fixed as https://bitbucket.org/squeaky/portable-pypy/downloads/pypy3.5-5.8-beta-linux_x86_64-portable.tar.bz2